src/components/PointCollection/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ScalerContext from '../../context/Scaler';
+import { createYScale } from '../../utils/scale-helpers';
+import Points from '../Points';
+import { seriesPropType, domainPropType } from '../../utils/proptypes';
+
+const propTypes = {
+  width: PropTypes.number.isRequired,
+  height: PropTypes.number.isRequired,
+  series: seriesPropType.isRequired,
+  domain: domainPropType.isRequired,
+  // (domain, width) => [number, number]
+  xScalerFactory: PropTypes.func.isRequired,
+};
+const defaultProps = {};
+
+const PointCollection = ({ width, height, series, domain, xScalerFactory }) => {
+  const xScale = xScalerFactory(domain, width);
+  const points = series.filter(s => !s.hidden).map(s => {
+    const yScale = createYScale(s.yDomain, height);
+    return <Points key={s.id} {...s} xScale={xScale} yScale={yScale} />;
+  });
+
+  return (
+    <g width={width} height={height}>
+      <clipPath id="scatterplot-clip-path">
```

Comment:
Should we also generate clipPath id based on series as of in `LineCollection`?

Reply:
Good catch; done.